frontend/app/routes/_gcweb-app.tsx

```diff
@@ -170,10 +170,10 @@ function PageFooter() {
 
   return (
     <footer id="wb-info">
-      <h2 className="wb-inv">{t('gcweb:footer.about-site')}</h2>
+      <h2 id='footer-heading' className="wb-inv">{t('gcweb:footer.about-site')}</h2>
       <div className="gc-sub-footer">
         <div className="d-flex align-items-center container">
-          <nav>
+          <nav aria-labelledby='footer-heading'>
```

Comment:
I think the `aria-labelledby` element's id should be the next `h3` and not the closest `h2`.